src/components/AddressAndSupporterInfo.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="main">
+      <div class="address">
+        <h3>Your new home is located at:</h3>
+        <p>{{address}}</p>
+      </div>
+      <div class="support">
+        <img class="personal-photo" alt="personal photo" v-bind:src="source">
+        <p>{{support}}</p>
+      </div>
+      <div class="btn-container">
+        <button class="btn">Reply to</button>
+        <button class="btn">Show jobs near me</button>
```

Comment:
try to use [Internationalization plugin for Vue.js](https://www.npmjs.com/package/vue-i18n) so your site would be easy to translate